README.md

```diff
@@ -325,6 +325,36 @@ To reimport just one entity, run:
 bundle exec rails dfe:analytics:import_entity[entity_name]
 ```
 
+
+## Event debugging
+
+If you wish to log events for debug purposes, create a file `config/analytics_event_debug.yml` containing an array of your event filters  under a `shared` key like:
+
+```yaml
+shared:
```

Comment:
@asatwal do we need the shared key or could event_filters be the top level?

Reply:
`shared` in this context means shared between environments. So this filter definition will be applicable to development, test and production.

Reply:
@asatwal that make sense. But in that case perhaps this behaviour should be documented if it isn't already?

Reply:
It's core Rails behaviour! Agree it's a bit funky.